mulighetsrommet-api/src/main/resources/db/migration/V198__refusjonskrav_betalingsinformasjon.sql

```diff
@@ -0,0 +1,5 @@
+alter table refusjonskrav
+    add column konto_nummer varchar(11) not null default '11111111111';
```

Comment:
Denne kolonnen må nok støtte å være `null` i stedet for en fiktiv default verdi. Vi kjenner ikke nødevendigvis til kontonummeret når refusjonskravet genereres opp, men burde kreve at det er tilgjengelig når refusjonskravet godkjennes av arrangør i stedet.